stl/inc/valarray

```diff
@@ -938,9 +938,16 @@ public:
 
     slice_array() = delete;
 
-    slice_array(const slice_array&); // not defined
+    slice_array(const slice_array&) = default;
 
-    slice_array& operator=(const slice_array&); // not defined
+    slice_array& operator=(const slice_array& _Right) {

```

Comment:
Pre-existing: the Standard says this `operator=` has a const qualifier ([\[slice.arr.assign\]](http://eel.is/c++draft/slice.arr.assign)).
```suggestion
    const slice_array& operator=(const slice_array& _Right) const {
```